dali/base/dautils.cpp

```diff
@@ -65,6 +65,29 @@ IPropertyTree * getDropZonePlane(const char * name)
     return getGlobalConfigSP()->getPropTree(xpath);
 }
 
+StringBuffer & makeDropZoneScope(StringBuffer & scope, bool isPlane, const char *dropZone, const char *path)
```

Comment:
this is _very_ similar to CDfsLogicalFilename::setExternal
and it supports validating the drop zone is a valid plane etc.

It would be better to alter CDfsLogicalFilename to support this.
i.e. introduce a setPlaneExternal(const char *planeName, const char *path);
which is almost same as the existing setExternal, can use common code.